Kinvey.Shared/Offline/PendingWriteAction.cs

```diff
@@ -53,8 +52,6 @@ public class PendingWriteAction
 		}
 
 		public NetworkRequest<T> toNetworkRequest <T>(AbstractClient client){
-			//T entity = cache.GetByIdAsync (entityId);
```

Comment:
Is there a point to keeping this function if all it does is return `null`?

Reply:
@vinaygahlawat 
No, there is not.
This is a "dead" code. We are going to have a special task for it.